lib/octocatalog-diff/catalog-diff/differ.rb

```diff
@@ -456,10 +457,18 @@ def ignored?(diff)
 
       # Cleanse parameters of filtered attributes.
       # @param parameters_hash [Hash] Hash of parameters
+      # @param sensitive_parameters [Array] Array of sensitive parameters
       # @return [Hash] Cleaned parameters hash (original input hash is not altered)
-      def cleanse_parameters_hash(parameters_hash)
+      def cleanse_parameters_hash(parameters_hash, sensitive_parameters)
         result = parameters_hash.dup
 
+        # hides sensitive params. We still need to know if there's a going to
+        # be a diff, so we hash the value.
+        sensitive_parameters.each do |p|
+          md5 = Digest::MD5.hexdigest result[p]
```

Comment:
To cope with Sensitive[Hash] parameters.  How about...
```suggestion
          md5 = Digest::MD5.hexdigest Marshal.dump(result[p])
```
?